src/client/components/vision/camera/view.tsx

```diff
@@ -72,6 +72,7 @@ export class CameraView extends Component<{ viewModel: CameraViewModel }> {
     const { viewModel } = this.props
     const renderer = viewModel.renderer(viewModel.canvas)
     if (renderer) {
+      renderer.setRenderTarget(null)
```

Comment:
I'm actually surprised it worked without this, but I believe this is the correct thing to do.

(and all goes away with my refactor)